pkg/blobstore/replication/metrics_blob_replicator.go

```diff
@@ -0,0 +1,162 @@
+package replication
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
+	"github.com/buildbarn/bb-storage/pkg/blobstore/slicing"
+	"github.com/buildbarn/bb-storage/pkg/clock"
+	"github.com/buildbarn/bb-storage/pkg/digest"
+	"github.com/buildbarn/bb-storage/pkg/util"
+	"github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc/codes"
```

Comment:
Please group imports by 2nd level domain, for consistency with the rest of the codebase. In other words, add an empty above this one.